lib/KCard/index.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+    :to="to"
+  >
+    <template #default>
```

Comment:
The logic in general is a bit confusing, there are quite a few v-ifs nested in various places and there is repeated code and there is no clear flow of what happens in each case.

I think we could use `display: flex`, and `flex-direction` to play with the layout a bit. Something to keep in mind is that the content will always be the same, what changes is the layout, so we could have something like:

```Vue
<BaseCard>
  <div class="display-flex" :style="wrapperStyle">
    <aside v-if="thumbnailDisplay!='none'">
      <KImg ... :style="imageStyle">
    </aside>
    <div>
      <slot #default>
    </div>
  </div>
  <div class="footer" v-if="$slots.footer">
    <slot #footer>
  </div>
</BaseCard>

...

<script>
...
wrapperSyle() {
  if (layout === "vertical") {
    return { flex-direction: "column" }
  }
  if (layout === "horizontal" && thumbnailDisplay === "small") {
    return { flex-direction: "row-reverse" } // To kep the image on the right
  }
  if (layout === "horizontal" && thumbnailDisplay === "large") {
    return { flex-direction: "row" } // To keep the image on the left
  }
}
</script>
```

In this way we do not repeat so many KImg, nor the content and we only play with the styles of the image, of the content depending on each case.